src/index.js

```diff
@@ -75,159 +61,109 @@ function App() {
-         * Component to the main path / until further instructions.
-         */}
-
         <PrivateRoute
           path="/dashboard"
-          redirect="/dashboard"
           allowRoles={[1, 2, 3, 4, 5]}
           component={TicketDashboard}
         />
-
         <PrivateRoute
           path="/memos"
-          redirect="/dashboard"
           allowRoles={[1, 2, 3, 4, 5]}
           component={Memos}
         />
-
         <PrivateRoute
           path="/memosform"
-          redirect="/dashboard"
           allowRoles={[1, 2, 3, 4, 5]}
           component={MemosForm}
         />
 
         <PrivateRoute
           path="/mymemos"
-          redirect="/dashboard"
           allowRoles={[1, 2, 3, 4, 5]}
           component={MyMemos}
         />
-
         <PrivateRoute
           path="/profile"
-          redirect="/login"
           allowRoles={[1, 2, 3, 4, 5]}
           component={Profile}
         />
-
         <PrivateRoute
           path="/users"
-          redirect="/dashboard"
           allowRoles={[1, 2, 5]}
           component={UserManagement}
         />
-
         <PrivateRoute
           path="/mentees"
-          redirect="/dashboard"
           allowRoles={[1, 2, 3, 5]}
           component={() => <div>"My Mentees" Component goes here</div>}
         />
-
         <PrivateRoute
           path="/resources"
-          redirect="/dashboard"
           allowRoles={[1, 2, 3, 4, 5]}
           component={ManageResources}
         />
-
         <PrivateRoute
           path="/attendance"
-          redirect="/dashboard"
           allowRoles={[1, 2, 5]}
           component={Attendance}
         />
-
         <PrivateRoute
           path="/support"
-          redirect="/dashboard"
           allowRoles={[1, 2, 3, 4, 5]}
           component={SupportRequests}
         />
-
         <PrivateRoute
           path="/meetings/schedule"
-          redirect="/dashboard"
           allowRoles={[1, 2, 3, 4, 5]}
           component={ScheduleMeeting}
         />
-
         <PrivateRoute
           path="/meetings"
-          redirect="/dashboard"
           allowRoles={[1, 2, 3, 4, 5]}
           component={ViewAllMeetings}
         />
-
         <PrivateRoute
           path="/calendar"
-          redirect="/dashboard"
           allowRoles={[1, 2, 3, 4, 5]}
           component={Calendar}
         />
-
         <PrivateRoute
           path="/matching"
-          redirect="/dashboard"
           allowRoles={[1, 5]}
           component={MentorMenteeMatching}
         />
-
         <PrivateRoute
           path="/applications"
-          redirect="/dashboard"
           allowRoles={[1, 2, 5]}
           component={PendingApplications}
         />
-
-        <PrivateRoute
-          path="/reviews"
-          redirect="/dashboard"
-          allowRoles={[1, 5]}
-          component={Reviews}
-        />
-
+        <PrivateRoute path="/reviews" allowRoles={[1, 5]} component={Reviews} />
```

Comment:
I'm not sure if we want to keep the formatting on this private route consistent with all the other ones over multiple lines for easier readability.